odata-protocol/11.4 Data Modification.md

```diff
@@ -13,6 +13,18 @@ the [`return`](#Preferencereturnrepresentationandreturnminimal) preference.
 A [success response](#SuccessResponses) indicates that data have been modified,
 regardless of whether the requested content could be returned.
 
+Data modification requests guarantee _atomicity_ in the following sense:
+When the request completes successfully, every subsequent request observes a state of the system
+in which either all or none of the changes have been carried out.
+But depending on how a service makes this guarantee, clients MAY observe
+a state of the system where the changes have been carried out only partially
+while the data modification request is still being executed.
+
+A transactional service is a service that protects against seeing such partial changes.
```

Comment:
Might have to have a better term than "Transactional" -- a transactional service may have different isolation levels.